### ISSUE TITLE
Stack Set Deployment Option for Region Concurrency

### ISSUE THREAD

**Issue by hitty5** [awslabs/aws-control-tower-customizations#60]
I'm using v2.0.0 mainly for rolling out manged AWS Config rules to almost all account in all regions. So in sum more than 200 stack instances are created. The stack set is created with following deployment options:
- Maximum concurrent accounts - 100
- Failure tolerance - 10

See https://github.com/awslabs/aws-control-tower-customizations/blob/677d12e16ecee9ae85c7fa056ded96aac5441525/source/aws/services/cloudformation.py#L28

So in case of so much instances with the above concurrent option it takes very long to apply the change to all accounts. Therefore it would be create to make the region concurrency configurable in manifest.yaml, e.g.

```
resources:
  - name: pcs-config-rules
    deploy_method: stack_set
    deploy_options:
       - region_concurrency = parallel 
```

Based on https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-concepts.html sequential deployment is the default selection

**Comment by groverlalit** [comment]
Thanks for raising this issue. This is already on our backlog and we plan to ship the fix in the next release.

**Comment by groverlalit** [comment]
This change was released in v2.1.0.
